.changeset/popular-seas-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': minor
+---
+
+Added new `CheckboxGroup` and `RadioGroup` components to group `Checkbox` and `Radio` components respectively.
```

Comment:
Sorry forgot to mention that for new components we usually add a bit more detail:

Something like:

( I can't make this a suggestion due to the code block syntax )

-- 


New `CheckboxGroup` and `RadioGroup` components to group `Checkbox` and `Radio` components are now available.

`CheckboxGroup` 

```jsx
<CheckboxGroup>
  <CheckboxGroup.Label>Choose your favorite features</CheckboxGroup.Label>
  <FormControl>
    <FormControl.Label>Actions notifications</FormControl.Label>
    <Checkbox value="actions" />
  </FormControl>
  <FormControl>
    <FormControl.Label>Packages</FormControl.Label>
    <Checkbox value="packages" />
  </FormControl>
  <FormControl>
    <FormControl.Label>Codespaces</FormControl.Label>
    <Checkbox value="codespaces" />
  </FormControl>
</CheckboxGroup>
```

:link: [Read the documentation for more `CheckboxGroup` examples](https://primer.style/brand/components/RadioGroup/react)


`RadioGroup`:

```jsx
<RadioGroup>
  <RadioGroup.Label>Choose your primary workspace</RadioGroup.Label>
  <FormControl>
    <FormControl.Label>Codespaces</FormControl.Label>
    <Radio name="workspace" value="codespaces" />
  </FormControl>
  <FormControl>
    <FormControl.Label>Local environment</FormControl.Label>
    <Radio name="workspace" value="local" />
  </FormControl>
  <FormControl>
    <FormControl.Label>Pen and paper</FormControl.Label>
    <Radio name="workspace" value="remote" />
  </FormControl>
</RadioGroup>
```

:link: [Read the documentation for more `RadioGroup` examples](https://primer.style/brand/components/RadioGroup/react)